engine/src/flutter/impeller/geometry/path_builder.cc

```diff
@@ -20,13 +20,16 @@ class RoundSuperellipseBuilder {
   typedef std::function<
       void(const Point&, const Point&, const Point&, const Point&)>
       CubicAdder;
+  typedef std::function<void(const Point&)> PointAdder;
```

Comment:
```suggestion
  using PointAdder = std::function<void(const Point&)>;
```

Reply:
I probably should have caught the previous one, but you can use `using` instead of typedef